README.md

```diff
@@ -26,14 +26,15 @@ If you're a web dapp developer, we've got two types of guides for you:
 
 ## Building locally
 
- - Install [Node.js](https://nodejs.org/en/) version 8.11.3 and npm version 6.1.0
-   - If you are using [nvm](https://github.com/creationix/nvm#installation) (recommended) running `nvm use` will automatically choose the right node version for you.
-   - Select npm 6.1.0: ```npm install -g npm@6.1.0```
- - Install dependencies: ```npm install```
- - Install gulp globally with `npm install -g gulp-cli`.
- - Build the project to the `./dist/` folder with `gulp build`.
- - Optionally, to rebuild on file changes, run `gulp dev`.
- - To package .zip files for distribution, run `gulp zip`, or run the full build & zip with `gulp dist`.
+- Install [Node.js](https://nodejs.org) version 8 and npm version 6
```

Comment:
latest version 8? or maybe 8.13+ ?

Just trying to avoid users running 8.11.3 or lower that might mess this up

Reply:
The Node version isn't too important, the app should work with any 8.y.z. The npm version is the real kicker and if they update the npm version for whatever version they're using they should be a-ok.

Reply:
That's fair. When I said "mess this up" I was referring to the package-lock.json getting massive changes if you're using different versions of npm 6. Maybe specifiy the latest npm version or 6.41.+ ?

Reply:
Oh yes that's a good point. I'll update this.